support-frontend/assets/pages/[countryGroupId]/checkout.tsx

```diff
@@ -698,6 +726,30 @@ function CheckoutComponent({ geoId, appConfig }: Props) {
 			}
 		}
 
+		if (
+			paymentMethod === 'StripeExpressCheckoutElement' &&
+			stripe &&
+			elements
+		) {
+			const { paymentMethod } = await stripe.createPaymentMethod({
+				elements,
+			});
+			if (!paymentMethod) {
+				return;
```

Comment:
I'm slightly lost in the diff here, what does this return do?

Reply:
Nice catch, this should be error handling which I have just added